spec/factories/story.rb

```diff
@@ -3,6 +3,7 @@
     association(:user)
     sequence(:title) {|n| "story title #{n}" }
     sequence(:url) {|n| "http://example.com/#{n}" }
+    sequence(:description) {|n| "#{n} Lorem ipsum dolor sit amet" }
```

Comment:
Remove this. If the test doesn't pass, it needs to explicitly create the `story` record it needs, not rely on an attribute of the global test setup.

Reply:
Removed.